docs/docs/how-tos/nebari-upgrade.md

```diff
@@ -6,7 +6,79 @@ description: An overview of how to upgrade Nebari. Includes instructions on upda
 
 # Upgrade Nebari
 
-This is a guide to upgrade Nebari to a newer version.  There are instructions for both upgrades from pre-0.4.0 to 0.4.0 as well as [minor upgrades](#minor-upgrade-to-nebari).
+This is a guide to upgrade Nebari to a newer version.  There are instructions for both minor upgrades as well as upgrades from [pre-0.4.0 to 0.4.0](#upgrade-nebari-from-pre-040-to-040).
+
+
+## Minor upgrade to Nebari
+
+### Backup existing data
+
+Perform manual backups of the NFS data and JupyterHub database.
+
+Always [backup your data](./manual-backup.md) before upgrading!
+
+### Install latest version of Nebari
+
+Use pip or conda to install the latest Nebari version.
+
+```shell
+pip install --upgrade nebari
+```
+or 
+
+```shell
+conda update nebari
+```
+
+
+### Update configuration file
+
+You may be deploying Nebari based on a local configuration file, or you may be using CI/CD workflows in GitHub or GitLab.  Either way, you will need to locate a copy of your `nebari-config.yaml` configuration file to upgrade it (and commit back to your git repo in the CI/CD case).
+ 
+For CI/CD deployments, you will need to `git clone <repo URL>` into a folder on your local machine if you haven't done so already.
+
+In the `nebari-config.yaml` file, update the latest version of Nebari.
+
+```yaml
+nebari_version: <version>
+```
+
+Next, its a good idea to validate your configuration file.  While not required, this is the best practice.
+
+```shell
+run nebari validate -c nebari-config.yaml
+```
+
+### Re-deploy Nebari
+
+If you are deploying Nebari from your local machine (see [this section](#cicd-render-and-commit-to-git) for CI/CD deployments), you will now have a `nebari-config.yaml` file that you can deploy.
+
+```shell
+nebari deploy -c nebari-config.yaml   
+```
+
+At this point you may see an error message saying that deployment is prevented due to the `prevent_deploy` setting in your YAML file. This is a safeguard to ensure that you only proceed if you are aware of possible breaking changes in the current upgrade.
+
+For example, we may be aware that you will lose data due to this upgrade, so need to note a specific upgrade process to keep your data safe. Always check the release notes of the release in this case and get in touch with us if you need assistance. For example, you may find that your existing cluster is intentionally deleted so that a new replacement can
+be deployed instead, in which case your data must be backed up so it can be restored after the upgrade.
+
+**When you are ready, remove the `prevent_deploy` setting from the config file, and run the `nebari deploy -c nebari-config.yaml` again.**
```

Comment:
This section should be moved to the `# Upgrade Nebari from pre-0.4.0 to 0.4.0` section since "normal" upgrades don't add this `prevent_key` to the config.

Reply:
✅